openmetadata-ui/src/main/resources/ui/src/rest/objectStoreAPI.ts

```diff
@@ -28,15 +28,18 @@ const configOptions = {
   headers: { 'Content-type': 'application/json' },
 };
 
-export const getContainers = async (
-  serviceName: string,
-  arrQueryFields: string | string[],
-  paging?: string
-) => {
+export const getContainers = async (args: {
+  service: string;
+  arrQueryFields: string | string[];
+  paging?: string;
+  root?: boolean;
+}) => {
+  const { service, arrQueryFields, paging, root } = args;
+
   const url = `${getURLWithQueryFields(
     `/containers`,
     arrQueryFields
-  )}&service=${serviceName}${paging ? paging : ''}`;
+  )}&service=${service}${paging ? paging : ''}&root=${root}`;
```

Comment:
In case root won't be available your URL will have it like:

`someurl/&root=undefined`

You can use qs package that we are already using

Axios also supports params you can use that as well instead making it in URL

Reply:
Thanks for catching it @chirag-madlani, I will make the changes.